content/en/entities/Status.md

```diff
@@ -296,6 +296,13 @@ aliases: [
 **Version history:**\
 3.5.0 - added
 
+### `quote` {#quote}
+
+**Description:** Information about the status being quoted, if any\
+**Type:* {{<nullable>}} [Quote]({{< relref "entities/quote" >}}), [ShallowQuote]({{< relref "entities/shallow_quote or null\
```

Comment:
When is it a `Quote` or a `ShallowQuote`? Should we define this somewhere?

Reply:
I am not sure we should. Apps should be able to handle both.